src/checkUsersTry.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+function checkUsersTry(usersTry, numberToGuess) {
+  if (isNaN(+usersTry) || usersTry.length !== 4) {
+    return 'The number must contain only 4 digits!';
```

Comment:
This error message doesn't work well with both conditions...
![image](https://user-images.githubusercontent.com/45945732/235765828-6b1859fa-70e9-49f2-b2ad-078f193aba38.png)
I'd split it to two messages:
`isNaN(usersTry)` --> `Please type a valid number using only digits`
`usersTry.length !== 4` --> `The number must contain only 4 digits!`